### PR TITLE
Update contributing documentation

### PR DESCRIPTION
**Pull request by contolini**
@Scotchester Please review when you return. Release management is now handled via [chat w/ hubot](https://github.com/cfpb/hubot-capital-framework). I also updated the contribution steps a l'il bit.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,15 +32,18 @@ For example, if you wanted to change `cf-buttons`, here's what you'd do:
...
-1. Describe the changes in the "Unreleased" section of `/CHANGELOG.md`.
-Prepend the component's name to the described changes.
-1. `git commit -am "Describe your commit"`
-1. `git push upstream doing-something-with-buttons`
-1. Go to https://github.com/cfpb/capital-framework and open a pull request to merge `doing-something-with-buttons` into `canary`.
+1. When you're done hacking, open `/CHANGELOG.md` and add a line item describing your changes in the "Unreleased" section.
+Use the format: `- **cf-component-name:** [MAJOR|MINOR|PATCH] Description of change`.
+[Here's an example](https://github.com/cfpb/capital-framework/pull/291/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR12).
+  - If the change affects *all* components, put `**all components:**` as the name.
+  - If the change affects CF and not any components (e.g. if you're just updating [README.md](README.md)), put `**capital framework**`.
```
Comment (Scotchester) [inline CONTRIBUTING.md]: `capital framework` or `capital-framework`? (See this PR's `CHANGELOG.md` update.)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,15 +32,18 @@ For example, if you wanted to change `cf-buttons`, here's what you'd do:
...
+1. `git commit -am "Fix button border radius"`
+1. `git push upstream button-fix`
```
Comment (Scotchester) [inline CONTRIBUTING.md]: Should be `push origin` if this was cloned from the CFPB source, as suggested in step 1 of this process.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -63,16 +66,7 @@ Now `~/Projects/owning-a-home/node_modules/cf-buttons` will be a symlink pointin
...
-1. Open `/CHANGELOG.md` and check the "Unreleased" section. For each component
-that's been modified, open its `/src/COMPONENT/package.json` and increment its
-version. Follow [SemVer](http://semver.org/) and increment either its major, minor,
-or patch version dependent upon the changes that were made to the component.
-1. Don't bump CF's primary `/package.json`. Travis will automatically increment its version.
-1. Delete any empty sections ("Added", "Changed", "Removed") from the "Unreleased" section.
-1. `git commit -am "Preparing release"`
-1. `git push upstream canary`
-1. Go to https://github.com/cfpb/capital-framework and open a pull request to merge `canary` into `master`.
+1. In a public chat room say "cfpbot capital framework prepare release"
```
Comment (Scotchester) [inline CONTRIBUTING.md]: Minor suggestion: "In a room where cfpbot is present (preferably the FEWD channel), say …"

File: CONTRIBUTING.md (hunk 1 above)
Comment (contolini) [inline CONTRIBUTING.md]: Good catch.

File: CONTRIBUTING.md (hunk 2 above)
Comment (contolini) [inline CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md (hunk 3 above)
Comment (contolini) [inline CONTRIBUTING.md]: 👍

**Comment by Scotchester** [comment]
This is amazing 😻

**Pull request merged**